index.bs

```diff
@@ -301,7 +301,7 @@ Perform Logger("warn", <var>data</var>).
   1. For each _item_ of _data_:
     1. Let _converted_ be a DOM tree representation of _item_ if possible, otherwise let _converted_ be an implementation-specific representation of _item_ judged to be maximally useful and informative.
     1. Append _converted_ to _finalList_.
-  1. Perform Printer("log", _finalList_).
+  1. Perform Printer("log", Formatter(_finalList_)).
```

Comment:
Good point, calling Formatter inside Printer is pretty much just calling Logger, will change. Yeah Formatter could probably be a little more rigorous